src/snyk/advisor/editor/editorDecorator.ts

```diff
@@ -0,0 +1,95 @@
+import { IConfiguration } from '../../common/configuration/configuration';
+import { getRenderOptions, updateDecorations } from '../../common/vscode/editorDecorator';
+import { HoverAdapter } from '../../common/vscode/hover';
+import { IVSCodeLanguages } from '../../common/vscode/languages';
+import { getMarkdownString } from '../../common/vscode/markdownString';
+import { IThemeColorAdapter } from '../../common/vscode/theme';
+import { DecorationOptions, Hover, TextEditorDecorationType } from '../../common/vscode/types';
+import { IVSCodeWindow } from '../../common/vscode/window';
+import { AdvisorScore } from '../advisorTypes';
+
+type LineDecorations = DecorationOptions[]; // array index is a line number
+const SCORE_PREFIX = 'Advisor Score';
```

Comment:
Let's have strings in the `messages` folder in the `src/snyk/advisor`. This would be consistent with others and is used to separate potentially translatable strings (data) from the logic code.